frontend/src/Admin/components/EditShowForm.js

```diff
@@ -0,0 +1,262 @@
+import React, { Component } from 'react'
```

Comment:
Probably not too difficult but it may be worth it to keep them separate now and investigate for refactoring later with the honor show around the corner.

Reply:
Yeah you’re right, we should keep the refactor in mind for the future though